app/serializers/note_serializer.rb

```diff
@@ -0,0 +1,6 @@
+class NoteSerializer < ActiveModel::Serializer
+  attributes :id, :title
+  attribute :note_type, key: :type
```

Comment:
A modo de comentario nada más, no me agrada el nombre de key `note_type`. Termina siendo redundante en el modelo `Note` que haya un atributo que sea `note_type`, ya se entiende que es el tipo de la nota. Debería ser `type` directamente. No obstante, es a modo comentario, ya está hecho así que dejémoslo ahí.

Reply:
> A modo de comentario nada más, no me agrada el nombre de key `note_type`. Termina siendo redundante en el modelo `Note` que haya un atributo que sea `note_type`, ya se entiende que es el tipo de la nota. Debería ser `type` directamente. No obstante, es a modo comentario, ya está hecho así que dejémoslo ahí.

El hecho de que lo hayamos nombrado `note_type`es porque type para Rails es una palabra reservada y se pone modo difícil con herencia (STI) y esas cosas 🤣 si te interesa leer sobre eso Valen --> 0is%20particularly%20useful%20when,is%20correctly%20identified%20and%20managed.